config/locales/candidate_interface/gcse_details.yml

```diff
@@ -30,6 +31,11 @@ en:
         You need at least a grade C.
 
         If you do not have this, contact teacher training providers to discuss your eligibility for their courses. You might be able to retake an exam or do an equivalence test.
+      international_baccalaureate_middle_years_programme: |
+        You normally need to have received at least 3 points for the subject in your International Baccalaureate Middle Year Programme, in order to demonstrate equivalence to a GCSE 4 (C) or above.
```

Comment:
```suggestion
        You normally need to have received at least 3 points in your International Baccalaureate Middle Year Programme, in order to demonstrate equivalence to a GCSE 4 (C) or above.
```

Reply:
Should we insert the subject here, to make it clear that we don't mean 3 points in the overall IB qualification (which you need a minimum 28 to get)?